sdk/examples/client/01_generate_addresses.rs

```diff
@@ -20,7 +20,7 @@ async fn main() -> Result<()> {
     dotenvy::dotenv().ok();
 
     for var in ["NODE_URL", "MNEMONIC"] {
-        std::env::var(var).unwrap_or_else(|_| panic!(".env variable '{var}' is undefined, see .env.example"));
+        std::env::var(var).expect(&format!(".env variable '{var}' is undefined, see .env.example"));
```

Comment:
not really sure I like it ... why not just?
```
`.unwrap_or_else(|| panic!(".env variable '{var}' is undefined, see .env.example"))
```
it's a bit longer, yes but wouldn't we wouldn't have to do the weird enable/disable lint :shrug:

Reply:
> why not just? .... it's a bit longer

This was the only reason to have less distracting code in the examples, don't think the lints are that weird

But you can fight with wild @kwek20 about what's better :trollface:

Reply:
I don't really have a strong opinion but I'm willing to do slightly unconventional things if it makes the examples a bit more straight to the point.